server/routes/contest.js

```diff
@@ -8,10 +8,12 @@ const {
   getContest,
   getContests,
   createSubmissionByContestId,
+  getUserContests,
 } = require('../controllers/contest');
 
 router.route('/').post(validateContest, createContest);
 router.route('/all').get(getContests);
 router.post('/:id/submission', protect, createSubmissionByContestId);
+router.route('/contests').get(protect, getUserContests);
```

Comment:
Maybe you can rename this route (for eg `user-contests`) for increased semantics?

Reply:
Done